cardano-strict-containers/src/Data/Maybe/Strict.hs

```diff
@@ -138,3 +153,26 @@ instance Alternative StrictMaybe where
   empty = SNothing
   SNothing <|> r = r
   l <|> _ = l
+
+instance Eq1 StrictMaybe where
```

Comment:
Could you please add a changelog for all new instances and bump up the minor version

Reply:
`1.5.0.0` does not seem to be released yet, so I didn't bump the minor version